Sources/Markdown/Utility/AtomicCounter.swift

```diff
@@ -8,7 +8,11 @@
  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-import CAtomic
+#if compiler(>=5.10)
```

Comment:
```suggestion
#if compiler(>=6.0)
```

Reply:
Thanks so much! I wasn't sure if it was fine, so thanks for verifying.